### PR TITLE
Add `text_input_auto_capitalization` modifier

### LINKED ISSUE #327: Modifier -> SwiftUI -> Text input and output: textInputAutocapitalization(_:)
Doc: https://developer.apple.com/documentation/swiftui/view/textinputautocapitalization(_:)

- [x] Swift implementation
- [x] Elixir implementation

### PR DESCRIPTION
**Pull request by dginsburg**
Closes #327

### CODE REVIEW

File: Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/TextInputAutocapitalizationModifier.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+/// ```
+///
+/// ## Arguments
+/// * ``autocapitalization``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+@available(iOS 16.0, watchOS 9.0, *)
+struct TextInputAutocapitalizationModifier: ViewModifier, Decodable, Equatable {
+    /// One of the capitalizing behaviors defined in the `TextInputAutocapitalization` struct or nil.
```
Comment (carson-katri) [inline Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/TextInputAutocapitalizationModifier.swift:25]: Can you list the possible values here?
```suggestion
    /// One of the capitalizing behaviors defined in the `TextInputAutocapitalization` struct or nil.
    ///
    /// Possible values:
    /// * `characters`
    /// * `sentences`
    /// * `words`
    /// * `never`
```

**Review by carson-katri — CHANGES_REQUESTED** [review]

File: Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/TextInputAutocapitalizationModifier.swift (hunk 1 above)
Comment (dginsburg) [reply Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/TextInputAutocapitalizationModifier.swift:25]: Of course I can and will do this moving forward. I was modeling my documentation from existing modifiers. If we want the documentation to be consistent about, well... enumerating the enums then there are some others to fix.

Comment (carson-katri) [reply Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/TextInputAutocapitalizationModifier.swift:29]: Yeah, I think its only important if we don't extend the type to be `Decodable` (in which case the cases should be listed on the extension's docs).

**Review by carson-katri — CHANGES_REQUESTED** [review]
You could probably remove `autocapitalization` from `TextFieldProtocol` now.

**Review by carson-katri — APPROVED** [review]

**Pull request merged**